app.js

```diff
@@ -1,9 +1,15 @@
 const express = require("express");
 const { getTopics } = require("./controllers/topics.controller");   
 const { errorHandler } = require("./error-handlers");
-
+const endpoints = require("./endpoints.json")
 const app = express();
 
+app.use(express.json());
+
+app.get("/api", (request, response, next) => {
+    response.status(200).send({ endpoints: endpoints })
```

Comment:
This could be written as { endpoints } using the shorthand as the key and the variable name are the same thing.